packages/graphback/src/utils/graphqlUtils.ts

```diff
@@ -34,3 +38,38 @@ export const filterObjectExtensions = (types: Type[]) => types.filter((t: Type)
 export const createImplementsInterfaceString = (names: string[]) => {
   return `implements ${names.map((name: string) => name).join(' & ')} `;
 }
+
+/**
+ * Collects all GraphQL files in a directory and reads the content into a string.
+ *
+ * @param schemaDir - The directory of the schema file(s)
+ */
+export const buildSchemaText = (schemaDir: string): string => {
```

Comment:
There can be some implications of this approach that will make validation hard. 
I think in the long run we can threat every file as individual migration. 

If for example, someone will copy the file, made some bogus modifications and forget about them then. 
The fact that we do not control how the models will be processed by migration means that there could be possible issues.